pygeos/test/test_misc.py

```diff
@@ -20,7 +20,12 @@ def test_version():
 
 def test_geos_version():
     expected = "{0:d}.{1:d}.{2:d}".format(*pygeos.geos_version)
-    assert pygeos.geos_version_string == expected
+    actual = pygeos.geos_version_string
+
+    # strip any beta / dev qualifiers
+    actual = actual.lower().rstrip("abcdefghijklmnopqrstuvwxyz")
```

Comment:
one suggestion is to use [string constants](https://docs.python.org/3/library/string.html#string.ascii_letters) via `from string import ascii_letters` at the top, then:
```suggestion
    actual = actual.rstrip(ascii_letters)
```
similar suggestion for below around line 46